ethstorage/p2p/protocol/tracker.go

```diff
@@ -46,16 +46,18 @@ type Tracker struct {
 	// from disk, which is linear in the number of items, but mostly constant
 	// in their sizes.
 	peerID   string
-	capacity float64 // TODO backward compatible, change to blob count in the next testnet
+	capacity float64
+	minValue float64
```

Comment:
It is confusing that capacity is the speed, but the minValue is actually a throughput value (speed * time). I suggest moving the minValue comparison to outsider logic (`getRequestSize`) because the tracker should not care about that.